### ISSUE TITLE
Bot Crashes in Channels with More Than 100 People

### ISSUE THREAD

**Issue by dannymichel** [z0mbieparade/b0t#26]
Not sure how you can test this, but the bot literally can't run in a channel with more than 100 people in it.

**Comment by oumike** [comment]
Testing this would be easy. Just write a basic script creating bot/users. What does the bot do?  Crash?  Not enter the room?

**Comment by dannymichel** [comment]
I think these bots would all need lfm accounts

**Comment by dannymichel** [comment]
I'm taking a break from IRC myself, but I wasn't aware z0mbie's IRC had 100+ people or bots with last.fm accounts associated.
If every tested under these conditions, I'd be interested in the results.

**Comment by oumike** [comment]
No. I'm saying you should create a room and setup 100+ simple irc bots (one script can generate multiple bots). Have one instance of this bot in the room. Register the bot army with the same lfm account. See if it crashes and what failed.

**Comment by dannymichel** [comment]
I'll look into that one day
